static/js/components/material/Drawer_test.js

```diff
@@ -89,25 +89,47 @@ describe("Drawer", () => {
     assert.isTrue(drawerNode.text().endsWith("Log out"))
   })
 
-  it("drawer element is rendered with collections list", async () => {
-    const wrapper = await renderDrawer()
-    ;[0, 1].forEach(function(col) {
-      const drawerNode = wrapper
-        .find(".mdc-list-item .mdc-list-item--activated")
-        .at(col)
-      assert.equal(drawerNode.text(), collections[col].title)
-      assert.equal(
-        drawerNode.props().href,
-        makeCollectionUrl(collections[col].key)
-      )
+  describe("when there are > 10 collections", () => {
+    beforeEach(() => {
+      const numCollections = 20
+      collections = [...Array(numCollections).keys()].map(() => makeCollection())
+      getCollectionsStub.returns(Promise.resolve({ results: collections }))
+    })
+
+    it("drawer element is rendered with max of 10 collections", async () => {
```

Comment:
maybe I'm missing it but this test doesn't seem to test that only the first 10 collections are rendered, maybe just add a simple check of the length of `wrapper.find(".mdc-list-item .mdc-list-item--activated")`?